website_product_document_download_counter/tests/product_document_test_utils.py

```
@@ -0,0 +1,29 @@
+"# -*- coding: utf-8 -*-"
```

Pylint (3.3.7)

[error] Unrecognized option found: readme-template-url, manifest-required-authors, manifest-required-keys, manifest-deprecated-keys, license-allowed, valid-odoo-versions

(E0015)